app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -67,17 +69,23 @@ class TopicInfoFragmentTest {
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
   fun testTopicInfoFragment_loadFragment_configurationChange_checkTopicName_isCorrect() {
     launchTopicActivityIntent(TEST_TOPIC_ID).use { scenario ->
       scenario.onActivity {  activity ->
         activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
       }
```

Comment:
Update this orientation change step with : `onView(ViewMatchers.isRoot()).perform(orientationLandscape())`